doc/sphinx/language/gallina-extensions.rst

```diff
@@ -1890,11 +1890,11 @@ This syntax extension is given in the following grammar:
 .. _explicit_app_grammar:
 
   .. productionlist:: explicit_apps
-       term     : @ `qualid` `term` … `term`
+       term2    : @ `qualid` `term` … `term2`
                 : @ `qualid`
                 : `qualid` `argument` … `argument`
-       argument : `term`
-                : (`ident` := `term`)
+       argument : `term2`
+                : (`ident` := `term2`)
```

Comment:
This should not stay like this. This grammar already appears in the main Gallina grammar. Two options:
- Remove the grammar from this section and talk describe it in the paragraph.
- Remove the corresponding rules from the main Gallina grammar and use `prodn +=` here.